docs/kyma/05-04-component-source-url.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Install components from user-defined URLs
+type: Configuration
+---
+
+The Kyma Operator allows you to use external URLs as sources for the components you decide to install Kyma with. Using this mechanism, you can install Kyma with one customized component, which you, for example, modified and store in GitHub or as a `.zip` or `.tgz` archive on a server, and have other components using the officially released sources.
```

Comment:
I have the impression that this sentence is too long and thus confusing. Can we split it into chunks to make it more digestive?